src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const SHORT_DAYS = 3;
+  const LONG_DAYS = 7;
+  const SHORT_DAYS_DISCOUNT = 20;
+  const LONG_DAYS_DISCOUNT = 50;
+  const RENTAL_PRICE = 40;
+  const TOTAL_PRICE = days * RENTAL_PRICE;
```

Comment:
It is customary to name static variables with a capital letter, if something is calculated in the variable, it is better to name it using the usual camelCase.